actors/runtime/src/test_utils.rs

```diff
@@ -65,6 +65,21 @@ lazy_static! {
         map.insert(*VERIFREG_ACTOR_CODE_ID, Type::VerifiedRegistry);
         map
     };
+    pub static ref ACTOR_CODES: BTreeMap<Type, Cid> = {
+        let mut map = BTreeMap::new();
+        map.insert(Type::System, *SYSTEM_ACTOR_CODE_ID);
+        map.insert(Type::Init, *INIT_ACTOR_CODE_ID);
+        map.insert(Type::Cron, *CRON_ACTOR_CODE_ID);
+        map.insert(Type::Account, *ACCOUNT_ACTOR_CODE_ID);
+        map.insert(Type::Power, *POWER_ACTOR_CODE_ID);
+        map.insert(Type::Miner, *MINER_ACTOR_CODE_ID);
+        map.insert(Type::Market, *MARKET_ACTOR_CODE_ID);
+        map.insert(Type::PaymentChannel, *PAYCH_ACTOR_CODE_ID);
+        map.insert(Type::Multisig, *MULTISIG_ACTOR_CODE_ID);
+        map.insert(Type::Reward, *REWARD_ACTOR_CODE_ID);
+        map.insert(Type::VerifiedRegistry, *VERIFREG_ACTOR_CODE_ID);
+        map
+    };
```

Comment:
Slightly more idiomatic:

```suggestion
    pub static ref ACTOR_CODES: BTreeMap<Type, Cid> = [
        (Type::System, *SYSTEM_ACTOR_CODE_ID),
        (Type::Init, *INIT_ACTOR_CODE_ID),
        (Type::Cron, *CRON_ACTOR_CODE_ID),
        (Type::Account, *ACCOUNT_ACTOR_CODE_ID),
        (Type::Power, *POWER_ACTOR_CODE_ID),
        (Type::Miner, *MINER_ACTOR_CODE_ID),
        (Type::Market, *MARKET_ACTOR_CODE_ID),
        (Type::PaymentChannel, *PAYCH_ACTOR_CODE_ID),
        (Type::Multisig, *MULTISIG_ACTOR_CODE_ID),
        (Type::Reward, *REWARD_ACTOR_CODE_ID),
        (Type::VerifiedRegistry, *VERIFREG_ACTOR_CODE_ID),
    ].into_iter().collect();
```

Reply:
When I try this I get "error[E0277]: a value of type `BTreeMap<Type, CidGeneric<64_usize>>` cannot be built from an iterator over elements of type `&(Type, CidGeneric<64_usize>)`"

Reply:
BtreeMap is having trouble implementing the right FromIterator method: " help: the trait `FromIterator<&(Type, CidGeneric<64_usize>)>` is not implemented for `BTreeMap<Type, CidGeneric<64_usize>>`
note: required by a bound in `collect`"